backend/program/media/item.py

```diff
@@ -116,6 +118,18 @@ def _determine_state(self):
             return States.Requested
         return States.Unknown
 
+    def clean_title(self, title: Optional[str]) -> Optional[str]:
```

Comment:
This is not needed as metadata we get is from Trakt. We dont need to sanitize it

Reply:
Solves the issue with Shōgun -> Shogun. Makes it easier to scrape with a title. 

Are you also able to take a look at downloading something with a weird title like "YU GI OH!" for me it caused issues with symlink folder creation if I remember correctly this solved.

Maybe that needs to be solved else where however in terms off general title scrapping when we don't have a IMDB cleaning it up does wonders regardless.

Reply:
oh, I was debating using a character translation map.. something like this

```py
translationTable = {
    'ā': 'a', 'ă': 'a', 'ą': 'a', 'ć': 'c', 'č': 'c', 'ç': 'c',
    'ĉ': 'c', 'ċ': 'c', 'ď': 'd', 'đ': 'd', 'è': 'e', 'é': 'e',
    'ê': 'e', 'ë': 'e', 'ē': 'e', 'ĕ': 'e', 'ę': 'e', 'ě': 'e',
    'ĝ': 'g', 'ğ': 'g', 'ġ': 'g', 'ģ': 'g', 'ĥ': 'h', 'î': 'i',
    'ï': 'i', 'ì': 'i', 'í': 'i', 'ī': 'i', 'ĩ': 'i', 'ĭ': 'i',
    'ı': 'i', 'ĵ': 'j', 'ķ': 'k', 'ĺ': 'l', 'ļ': 'l', 'ł': 'l',
    'ń': 'n', 'ň': 'n', 'ñ': 'n', 'ņ': 'n', 'ŉ': 'n', 'ó': 'o',
    'ô': 'o', 'õ': 'o', 'ö': 'o', 'ø': 'o', 'ō': 'o', 'ő': 'o',
    'œ': 'oe', 'ŕ': 'r', 'ř': 'r', 'ŗ': 'r', 'š': 's', 'ş': 's',
    'ś': 's', 'ș': 's', 'ß': 'ss', 'ť': 't', 'ţ': 't', 'ū': 'u',
    'ŭ': 'u', 'ũ': 'u', 'û': 'u', 'ü': 'u', 'ù': 'u', 'ú': 'u',
    'ų': 'u', 'ű': 'u', 'ŵ': 'w', 'ý': 'y', 'ÿ': 'y', 'ŷ': 'y',
    'ž': 'z', 'ż': 'z', 'ź': 'z', 'æ': 'ae', 'ǎ': 'a', 'ǧ': 'g',
    'ə': 'e', 'ƒ': 'f', 'ǐ': 'i', 'ǒ': 'o', 'ǔ': 'u', 'ǚ': 'u',
    'ǜ': 'u', 'ǹ': 'n', 'ǻ': 'a', 'ǽ': 'ae', 'ǿ': 'o',
}
translationTable = str.maketrans(translationTable)

def translate(title: str):
    return title.translate(translationTable)
```

Reply:
Ohhhh nice, turns out there's a library that already does it called UniDecode if I remember correctly, its one off the first lines in that function haven't tested it to much yet. Works for Shogun however